src/scripts/config/config.go

```diff
@@ -3,6 +3,8 @@ package config
 import (
```

Comment:
the imports are a little out of whack here (since we've not yet had a chance to plug in linting / formatting tools)

```suggestion
import (
	"bytes"
	"fmt"
	"os"
	"runtime"
	"strings"

	"github.com/a8m/envsubst"
	"github.com/joho/godotenv"

	"github.com/CircleCI-Public/slack-orb-go/src/scripts/ioutils"
```